docs/apm/service-maps.asciidoc

```diff
@@ -0,0 +1,50 @@
+[[service-maps]]
+=== Service maps
+
+beta::[]
+
+A service map is a real-time diagram of the interactions occurring in your application’s architecture.
+It allows you to easily visualize data flow, and high-level statistics, like average transaction duration,
```

Comment:
This sentence is a bit long and the part after the last comma doesn't really make sense. I would tweak it some:

> It allows you to easily visualize data flow and high-level statistics like average transaction duration, requests per minute, errors per minute, and metrics, allowing you to quickly assess the status of services.

Reply:
Great fix